test/integration/elasticquota_controller_test.go

```diff
@@ -0,0 +1,359 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
```

Comment:
Please ensure the empty lines are placed properly. (L7 and L9)

Reply:
👍
Thank you for your detailed comments.
I feel like I learned a lot of new things.